core/controllers/profile.py

```diff
@@ -309,6 +425,44 @@ def get(self):
 class SignupHandler(base.BaseHandler):
     """Provides data for the editor prerequisites page."""
 
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {},
+        'POST': {
+            'username': {
+                'schema': {
+                    'type': 'basestring',
+                    'validators': [{
+                        'id': 'is_valid_username_string'
+                    }]
+                },
+                'default_value': None
```

Comment:
Why `None`?

Reply:
From a few requests, I also found it as an optional arg. That's why I added `none` as `default_value`. Please tell me if I did wrong.

Reply:
I don't see the above.

Reply:
Actually, I found that it is optional in the request, that's why I added None as the default value.

Reply:
I what request? In the request coming from frontend?

Reply:
same as below.